test/test_quantized.py

```diff
@@ -613,6 +613,89 @@ def test_qconv(
         # Make sure the results match
         np.testing.assert_equal(result_q, Y_q.int_repr().numpy())
 
+class TestQNNPackOps(TestCase):
+
+    """Tests the correctness of the quantized::qnnpack_relu op."""
+    @given(Q=qtensor(shapes=array_shapes(1, 5, 1, 5), dtypes=((torch.quint8, np.uint8, 0),),))
+    def test_qnnpack_relu(self, Q):
+        X, (scale, zero_point), (qmin, qmax), (torch_type, np_type) = Q
+        relu = torch.ops.quantized.qnnpack_relu
+
+        Y = X.copy()
+        X = torch.from_numpy(X)
+
+        qX = torch.quantize_linear(X, scale=scale, zero_point=zero_point,
+                                   dtype=torch_type)
+        qY_hat = relu(qX)
+
+        Y[Y < 0] = 0
+        qY = _quantize(Y, scale, zero_point, dtype=np_type)
+        np.testing.assert_equal(qY, qY_hat.int_repr())
+
+    """Tests the correctness of the quantized::qnnpack_fc op."""
+    def test_qnnpack_fc(self):
```

Comment:
why don't do hypothesis testing too?

also, it would be nice to test comparison (approximately) with the float version of Linear

Reply:
Please ensure that 
1. qnnpack tests are identical to tests for fbgemm ops (same level of error with respect to float/reference quantized implementation)

Also, add a test to ensure that error between qnnpack and fbgemm quantized ops is small. (quantized implementations are consistent)

Reply:
Done. Added hypothesis testing and also tested against FP32 results of nn.Linear